Spatial.tscn

```diff
@@ -600,125 +601,158 @@ disabled = true
 text = "Reverse Path Direction"
 
 [node name="SettingsDialog" type="WindowDialog" parent="Control/Dialogs"]
+visible = true
```

Comment:
This should be false. Godot gives a node configuration warning that pop ups will be hidden on run time

Reply:
accepting suggestion to remove the line, it will be re-added when saving is recreated.